pkg/skaffold/build/app.tar

```diff
@@ -0,0 +1 @@
+bazel:app
```

Comment:
I think we shouldn't commit this file?

Reply:
Sorry, this was added accidentally.  Removed now, thanks!